www/js/controller.js

```diff
@@ -477,7 +478,20 @@ module.exports = {
     global.platform.ipc.send('clear-apv');
   },
 
-  sendLine(shabadID, lineID, Line, rows) {
+  remapLine(rawLine) {
+    const Line = Object.assign(rawLine, {});
```

Comment:
`Object.assign({}, rawLine);`

Reply:
I am not sure why but that didn't work for me. 🙏 🤔 

Looking into it again.